package.json

```diff
@@ -16,6 +16,7 @@
     "coverage": "vitest run --coverage",
     "e2e": "cypress open",
     "lint": "stylelint \"**/*.{vue,scss}\" && prettier . --check && eslint --ext ts,js,vue,json . && ELECTRON=true nuxi typecheck",
+    "lint:fix": "stylelint --fix \"**/*.{vue,scss}\" && prettier . --check --write && eslint --fix --ext ts,js,vue,json .",
```

Comment:
Could you please check first what combining the options `--check` and `--write` does to the prettier command?

Reply:
I have removed the option `--check` since it is useless for lint fix